lua/weapons/weapon_snowball.lua

```diff
@@ -0,0 +1,133 @@
+SWEP.PrintName = "Snowballs"	
+
+SWEP.Instructions = "Left click to throw a snowball\nRight click changes trail color"
+
+SWEP.ViewModel = "models/weapons/v_snowball.mdl"
+SWEP.WorldModel = "models/weapons/w_snowball.mdl" 
+
+SWEP.Slot = 4
+SWEP.SlotPos = 1
+
+SWEP.DrawAmmo = false
+SWEP.DrawCrosshair = false
+SWEP.ViewModelFOV = 68
+
+SWEP.AutoSwitchTo = false
+SWEP.HoldType = "grenade"
+
+SWEP.Category = "Snowballs"
+SWEP.Spawnable = true
+
+SWEP.Primary.ClipSize = 1
+
+SWEP.Primary.Automatic = true
+SWEP.ThrowSound = Sound("weapons/iceaxe/iceaxe_swing1.wav")
```

Comment:
missing sound?

Reply:
It's one of the Half Life 2 sounds (the crowbar swing sound, specifically) so it should be fine